ENIGMAsystem/SHELL/Universal_System/Extensions/ExternalFuncs/POSIX/ExternalFuncs.h

```diff
@@ -0,0 +1,5 @@
+#include <dlfcn.h>
```

Comment:
The files `POSIX/ExternalFuncs.h` and `Win32/ExternalFuncs.h` can be combined if you just move these includes into the sources. Please do this.